### PR TITLE
Registering and tagging a package for parallel MCMC

### PR DESCRIPTION
**Pull request by KrisDM**
See https://quantifyinguncertainty.github.io

### CODE REVIEW

File: GeneralizedMetropolisHastings/versions/0.0.1/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+a71d8f533d19f9add37b4df690978d9f045706e3
```
Comment (tkelman) [inline GeneralizedMetropolisHastings/versions/0.0.1/sha1]: can you push the git tag so it shows up at https://github.com/QuantifyingUncertainty/GeneralizedMetropolisHastings.jl/releases ?

Comment (tkelman) [inline GeneralizedMetropolisHastings/versions/0.0.1/sha1]: ping ^^

Comment (KrisDM) [inline GeneralizedMetropolisHastings/versions/0.0.1/sha1]: Thanks, Tony, and apologies for the delay. I was on holiday but just returned. I've pushed a tag on the repository itself.

Comment (tkelman) [inline GeneralizedMetropolisHastings/versions/0.0.1/sha1]: the tag you pushed is a different commit than you've listed here

**Comment by KrisDM** [comment]
Made a mess, starting over...

**Pull request closed without merging**